packages/adp-tooling/src/types.ts

```diff
@@ -192,6 +213,14 @@ export const enum ChangeType {
     CHANGE_INBOUND = 'appdescr_app_changeInbound'
 }
 
+/**
+ * Enumerates the types of projects that can be made, each representing a specific kind of project structure.
+ */
+export const enum ProjectType {
```

Comment:
Both are S/4 systems here, so please change it to something like `Cloud` and `OnPremise`. Even better would be if you reuse the type `OperationsType` (https://github.com/SAP/open-ux-tools/blob/main/packages/axios-extension/src/abap/types/adt-types.ts#L77) because that is exactly what you will get back from the backend as identify during the prompting phase.

Reply:
Done